chia/util/streamable.py

```diff
@@ -439,6 +442,10 @@ def function_to_parse_one_item(f_type: type[Any]) -> ParseFunctionType:
         key_parse_inner_type_f = function_to_parse_one_item(inner_types[0])
         value_parse_inner_type_f = function_to_parse_one_item(inner_types[1])
         return lambda f: parse_dict(f, key_parse_inner_type_f, value_parse_inner_type_f)
+    if is_type_Enum(f_type):
+        if not hasattr(f_type, "_streamable_proxy"):
```

Comment:
i haven't dug in, but could just requiring this info be available in all cases let us avoid `hasattr()` etc calls?  like just say a streamable will have proxy information and it will be `None` in non-enum cases and non-`None` in enum cases?  and maybe the info is help wherever streamable info is already held instead of in a new attribute.